### PR TITLE
Feature/framer motion

### PR DESCRIPTION
**Pull request by tonycarpenter21**
What (if any) features are you implementing?
- Motion animations

What (if anything) did you refactor?
- Nothing

Were there any issues that arose?
- No

Is there anything that you need from your teammate?
- No

Any other comments, questions, or concerns?
- No

### CODE REVIEW

File: src/ArtDetails/ArtDetails.tsx [hunk 1]
```diff
@@ -3,12 +3,18 @@ import './ArtDetails.scss';
...
   const likedMessage = props.isLiked ? <p className="percent-message">You and {Math.floor(Math.random() * 100) + 1 + '%'} of other voters liked this art! You have great taste!</p>
                                      : <p className="percent-message">You and {Math.floor(Math.random() * 100) + 1 + '%'} of other people agree! Congratulations!</p>
   return (
-    <section className="art-details-container">
+    <motion.section 
+      className="art-details-container"
+      animate={{ opacity: 1 }}
+      initial={{ opacity: 0 }}
+      transition={{ duration: .4 }}
+    >
```
Comment (StevieAmb) [inline src/ArtDetails/ArtDetails.tsx]: This is awesome! The fade-in is awesome!

Comment (mr-ryan12) [inline src/ArtDetails/ArtDetails.tsx]: Really great addition to our application! Thank you for implementing this feature!

File: src/Categories/Categories.tsx [hunk 2]
```diff
@@ -11,7 +12,11 @@ const Categories: React.FC<CategoriesProps> = props => {
   })
   
   return (
-    <section className="main-page-container">
+    <motion.section
+      className="main-page-container"
+      animate={{ opacity: [0 ,1] }}
+      transition={{ duration: .4 }}
+    >
```
Comment (mr-ryan12) [inline src/Categories/Categories.tsx:19]: Awesome work!

File: src/Categories/Categories.tsx [hunk 3]
```diff
@@ -20,7 +25,7 @@ const Categories: React.FC<CategoriesProps> = props => {
       <section className="categories-container">
         {allCategories}
       </section>
-    </section>
+    </motion.section>
```
Comment (StevieAmb) [inline src/Categories/Categories.tsx:28]: It's awesome you took the time to go learn this feature! Nice work!

**Review by StevieAmb — APPROVED** [review]

**Pull request merged**